added publisher read only mode docs (#5098)
wso2/docs-apim: en/docs/get-started/about-this-release.md

	 Authentication for Prototype APIs is available by default so that the client apps can also be created using authentication to test out the APIs. If needed you can [disable authentication for Prototype APIs]({{base_path}}/design/prototype-api/create-a-prototype-api/)) at the resource level or API level.

- **Publisher Read Only Mode (M3)** 
    

>      A user who has view/read only permissions can only view the API/Services details in the publisher portal.He/She should not be able to edit/create/publish APIs/Services.

Reviewer: ```suggestion
     A user who has view/read only permissions can only view the API/Services details in the Publisher portal. Such users should not be able to edit/create/publish APIs/Services.
```
